app/Models/Mship/Feedback/Feedback.php

```diff
@@ -82,6 +86,11 @@ public function scopeUnActioned($query)
         return $query->where('actioned_at', null);
     }
 
+    public function scopeSent($query)
+    {
+        return $query->where('sent_at', '!=', null);
```

Comment:
Could shorten this to `whereNotNull('sent_at')`. Could refactor that above as well if you find yourself doing nothing...

Reply:
4ecfad7